site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java

```diff
@@ -332,7 +332,7 @@ public SecurityAdvice isAllowed(String userId, String function, String reference
 			"-siteInfo-importUser",
 			"-uploadArchive",
 			"-siteInfo-changeAdmin", // 63
-			"-selectAdmin", // 64
```

Comment:
Doesn't look right, there's lots of existing references to 64 in the file that shouldn't be `exportMemberList`